nvme-print.c

```diff
@@ -2934,9 +2940,38 @@ void json_nvme_zns_id_ns(struct nvme_zns_id_ns *ns,
 	json_free_object(root);
 }
 
+static void show_nvme_id_ns_zoned_zoc(__le16 ns_zoc)
+{
+	__u16 zoc = le16_to_cpu(ns_zoc);
+	__u8 rsvd = (zoc & 0xfc) >> 2;
+	__u8 ze = (zoc & 0x2) >> 1;
+	__u8 vzc = zoc & 0x1;
+	if (rsvd)
+		printf("  [7:2] : %#x\tReserved\n", rsvd);
+	printf("  [1:1] : %#x\tZone Excursions are %sEnabled\n",
```

Comment:
Zone Active Excursions

Reply:
How about "Zone Active Excursions management is required"?

Reply:
@MatiasBjorling 
when its set to ‘1’, then a controller may transition the state. so I don't think required is right.
if this mentioned as required, it seems user need to do something for it.

Reply:
Agree - I noodled over the weekend. I think it should be:
Zone Active Excursions: Yes (Host support required) / No
Variable Zone Capacity: Yes (Host support required) / No